electron_app/src/clientManager.js

```diff
@@ -95,13 +95,20 @@ const checkExpiredSession = async (
   const SUSPENDED_ACCOUNT_REQ_STATUS = 451;
 
   const status = requirementResponse.status;
-  const { recipientId: accountRecipientId } = requestparams;
-  const [{ recipientId, refreshToken }] = accountRecipientId
+  const accountRecipientId =
+    typeof requestparams === 'object'
+      ? requestparams.recipientId
+      : requestparams;
```

Comment:
```suggestion
      : null;
```

Reply:
if requestParams is not an object, it might be a string `recipientId`, for example in `getUserSettings`, so leaving it as is, is ok